### DISCUSSION TITLE
[@types/node] `AbortController.abort` can receive an optional `reason` parameter

### DISCUSSION THREAD

**Discussion by tonivj5** [DefinitelyTyped/DefinitelyTyped#64570] [Issues with a @types package]
The abort method should receive an optional parameter, currently if you use the same code with web & node, typechecking on __node__ complain

```sh
TS2554: Expected 0 arguments, but got 1.
    14 |   const ac = new AbortController();
    15 |
  > 16 |  ac.abort('reason');
       |           ^^^^^^^^
```

I've found this issue https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/58273, but typescript added this optional parameter to its `dom.d.ts`: https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L2046
 
Refs:
https://nodejs.org/api/globals.html#abortcontrollerabortreason
https://nodejs.org/api/globals.html#abortsignalreason
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b92231c3ef87e84860ea674ec38d8e694e5dc64e/types/node/globals.d.ts#L53-L56

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
`reason` is optional so can be safely added.
